modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss

```diff
@@ -7,4 +7,20 @@
 // acceptable amount of bloat for maintaining styles automatically.
 body {
   @extend .su-wysiwyg-text;
+
+  a[href^="mailto:"] {
+    &::after {
+      @include padding(null null null 0.1rem);
+      content: "\f0e0";
+      color: $su-color-bright-blue;
+      font-family: "Font Awesome 5 Free";
```

Comment:
I don't believe Font Awesome styles are added in the wysiwyg. We can probably get away with just a simple png for the wysiwyg.

Reply:
Ahhh! Thank you! I'll give it a go